solaris/solaris11/SPECS/template_agent_v3.10.0.json

```diff
@@ -664,22 +664,6 @@
         "type": "file",
         "user": "root"
     },
-    "/var/ossec/lib": {
-        "class": "static",
-        "group": "ossec",
-        "mode": "0750",
-        "prot": "drwxr-x---",
-        "type": "directory",
-        "user": "root"
-    },
-    "/var/ossec/lib/libwazuhext.so": {
```

Comment:
If the `libwazuhext.so` is not build, the `/var/ossec/lib` directory still exists if you install from sources, so you don't need to delete it.